content/platform/swarm-learning/home.md

```diff
@@ -0,0 +1,66 @@
+  Learning framework. "
+image: /img/platforms/guid-e80d248e-e754-498e-99d6-67508092f779-high.jpg
+width: large
+priority: 0
+frontpage: false
+active: false
+---
+HPE Swarm Learning is a decentralized, privacy-preserving Machine Learning framework. This framework utilizes the computing power at, or near, the distributed data sources to run the Machine Learning algorithms that train the models. It uses the security of a blockchain platform to share learnings with peers in a safe and secure manner. In  HPE Swarm Learning, training of the model occurs at the edge, where data is most recent, and where prompt, data-driven decisions are mostly necessary. In this completely decentralized architecture, only the insights learned are shared with the collaborating ML peers, not the raw data. This tremendously enhances data security and privacy.
+
+- - -
+
+## Articles
+
+* **[2﻿021 Nature Paper - Swarm Learning for decentralized and confidential clinical machine learning](https://www.nature.com/articles/s41586-021-03583-3)**
+* **[HPE Labs - Swarm Learning: Turn your distributed data into competitive edge](https://www.hpe.com/psnow/doc/a50000344enw?jumpid=in_lit-psnow-red)**
+* **[Research gate - Swarm Learning as a privacy-preserving machine learning approach for disease classification](https://www.researchgate.net/publication/342495847_Swarm_Learning_as_a_privacy-preserving_machine_learning_approach_for_disease_classification)**
+
+- - -
+
+## Learn more about HPE Swarm Learning and its underlying technology
+
+**\-﻿ What's All the Buzz about Swarm Learning?**
+
+In this video, Dr. Eng Lim Goh highlights the challenges of data privacy in the organizations to use siloed data to be used to bring out collective insights. Also, he explains HPE Swarm learning has the potential to change our lives and solve some of the world's most pressing and complex problems.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JqwjyENd89I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**\-﻿ The Big Shift: What is Swarm Learning?**
+
+Swarm learning is the next gold rush for machine intelligence - training at the edge so the edge devices get smarter and also train their peers. With no central authority, Blockchain is integrated to add control, privacy, and security.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6Fep6Lw5t-U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**\-﻿ HPE Swarm Learning Product Introduction**
+
+HPE Swarm Learning – industry’s first privacy preserving, decentralized machine learning solution . Developed by HPE Labs, HPE Swarm Learning enables AI/ML model developers to increase accuracy and reduces biases in their models.
```

Comment:
This first phrase... should it actually be a header? It's not a complete sentence... that's why I'm asking. But you do have a period towards the end, so maybe you do mean it as a sentence. In that case, it should read HPE Swarm Learning is the industry's first privacy preserving, decentralized machine learning solution." (remove that extra space)

In the second sentence, the last phrase should read "... enables AI/ML model developers to increase accuracy and **reduce** biases..." ["developers" is plural]

Reply:
done